src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -6,11 +6,14 @@
     border-top-left-radius: 4px;
     bottom: 0;
     display: none;
-    height: 30px;
-    padding: 6px 6px 0;
+    height: 36px;
+    padding: 6px;
     position: fixed;
     right: 0;
     z-index: 99999;
+    box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
```

Comment:
prefixed versions must be before the standard one, so that the standard version wins over the prefixed one